apps/web/build/generateScssVariables.ts

```diff
@@ -0,0 +1,70 @@
+import fs from 'node:fs';
+import path from 'node:path';
+import { getTailwindColorsOklch } from '../utils/tailwindHelper';
+import type { TailwindColors } from '~/composables/useConfigurationDrawer';
+import { rgb } from 'culori';
+import { formatRgb } from 'culori/fn';
+
+const primaryColor = process.env.PRIMARY || '#0c7992';
+const secondaryColor = process.env.SECONDARY || '#008ebd';
+
+export const oklchToRgb = (oklch: string) => {
```

Comment:
```suggestion
const oklchToRgb = (oklch: string) => {
```
**todo**: Not used elsewhere.